.github/workflows/build-dumb-pypi.yml

```diff
@@ -0,0 +1,177 @@
+name: Build dumb PyPI on GH pages
+
+on:
+  push:
```

Comment:
note for myself: this should be limited to master

Reply:
main